src/common/helper.js

```diff
@@ -394,7 +392,7 @@ async function getUserById (userId, enrich) {
  * @returns the request result
  */
 async function createUbhanUser ({ handle, firstName, lastName }) {
```

Comment:
@imcaizheng here is a typo `createUbhanUser` => `createUbahnUser`

Reply:
never mind, this is existent code

Reply:
@maxceem Fixed.